### DISCUSSION TITLE
在使用iconfront的时候遇到了问题

### DISCUSSION THREAD

**Discussion by Orangenazuna** [theme-shoka-x/hexo-theme-shokaX#252] [Q&A] [answered]
按照技术文档中的步骤，我引入Iconfont并且创建了自己的项目。在我新加入了两个icon并且使用它们的时候，我发现网页并没有显示，控制台也没有报错。
这是我的代码`//at.alicdn.com/t/c/font_4447313_46osgyfwzh4.css`。
我也在/_config.shokaX.yml文件中，加入了iconfont: "4447313_46osgyfwzh4"语句，并且修改了/source/css/_iconfont.styl文件。

**Comment by FidStyle** [comment]
1. 使用的shokaX版本是什么
2. 能提供截图或者网站吗

看了下生成的css代码是没有问题的
@Orangenazuna

**Reply by FidStyle** [reply]
配置文件引入了相应的部分吗，比如`social:`中
`  gitee: https://gitee.com/joytion || gitee || "#e60026"`

**Reply by Orangenazuna** [reply]
是的，
github: https://github.com/Orangenazuna?tab=repositories || github || "#191717"
steam : https://steamcommunity.com/profiles/76561198138360316/ || steam || "#f7f7f2"
配置文件中我也引入了，但除了默认iconfront的图标能显示，新添加的都显示不了

**Reply by FidStyle** [reply]
你的css项目我看看，应该是font的问题，邀请我为只读成员吧

**Reply by FidStyle** [reply]
可能跟图标需要为灰度（无颜色）图有关

**Reply by FidStyle** [reply]
@Orangenazuna

**Reply by Orangenazuna** [reply]
我这个hexo d目前有bug，我在研究怎么解决T_T，静态网页传不上git

**Reply by FidStyle** [reply]
我需要邀请链接（）
这个css看不了内容的

**Reply by FidStyle** [reply]
![image](https://github.com/theme-shoka-x/hexo-theme-shokaX/assets/115857087/731b4bf2-2ad4-4279-ae25-31f032bcccf1)
邀请我为只读成员即可，我看看怎么个事
@Orangenazuna

**Reply by FidStyle** [reply]
hexo d的问题建议google，猜测为git的问题

**Reply by Orangenazuna** [reply]
https://www.iconfont.cn/invite?type=project&token=zE25nPc8dqLITf6A#邀请你加入「MyblogIcon」

**Reply by FidStyle** [reply]
![image](https://github.com/theme-shoka-x/hexo-theme-shokaX/assets/115857087/8b931aa7-0a9e-4675-9c1c-14c6911c8fe3)
我这儿没问题

**Reply by FidStyle** [reply]
![image](https://github.com/theme-shoka-x/hexo-theme-shokaX/assets/115857087/d0c10c50-7e6c-423c-aa57-aef189c169d9)
这里设置了吗

**Reply by Orangenazuna** [reply]
![image](https://github.com/theme-shoka-x/hexo-theme-shokaX/assets/130027357/b1ba906f-54ab-44c5-8497-883ff119a96a)
已经设置好了，咦，那是怎么回事呢

**Reply by FidStyle** [reply]
你的项目也没问题（），我刚又试了一下

**Reply by Orangenazuna** [reply]
在_config.shokaX.yml 文件中写了iconfont: "4447313_46osgyfwzh4"
并且source\css\_iconfront.styl 里我也复制进去了

**Reply by FidStyle** [reply]
![image](https://github.com/theme-shoka-x/hexo-theme-shokaX/assets/115857087/79e46b2f-416f-49b5-a650-affbf1de7310)
有没有可能是颜色原因（）
闹乌龙了

**Reply by FidStyle** [reply]
"#e60026"正常显示为红色，你那个太浅了？
@Orangenazuna

**Reply by Orangenazuna** [reply]
![image](https://github.com/theme-shoka-x/hexo-theme-shokaX/assets/130027357/247040b3-8e61-41d9-9d8d-b311013307c4)

**Reply by FidStyle** [reply]
改完记得重新hexo s，这个不会即时刷新的

**Reply by Orangenazuna** [reply]
![image](https://github.com/theme-shoka-x/hexo-theme-shokaX/assets/130027357/fa6930c2-2008-4583-bbbb-688a831ec87a)
改成红色了，但是看起来没有icon

**Reply by FidStyle** [reply]
![image](https://github.com/theme-shoka-x/hexo-theme-shokaX/assets/115857087/9076f72d-ea00-44e4-bd44-761a4d6bc641)
改完记得先重新hexo s再刷新（）
`hexo s`和`刷新`缺一不可

**Reply by FidStyle** [reply]
![image](https://github.com/theme-shoka-x/hexo-theme-shokaX/assets/115857087/80faaa5d-4462-4050-b913-697c40d3e7d6)
就改了三个地方，用的是你的原始项目

**Reply by Orangenazuna** [reply]
![image](https://github.com/theme-shoka-x/hexo-theme-shokaX/assets/130027357/93d7fe1e-dc23-472a-80e2-1cffdd13b078)
![image](https://github.com/theme-shoka-x/hexo-theme-shokaX/assets/130027357/201a04a2-e0c7-4a18-bfd4-79847a90c511)
![image](https://github.com/theme-shoka-x/hexo-theme-shokaX/assets/130027357/ecc8d0eb-fda8-4b6f-8478-9eee4174b0fb)
会不会是这个iconfront.styl里我这么写不对

**Reply by FidStyle** [reply]
项目方便打包发我邮箱吗（）
joytion@foxmail.com
图片看起来没啥问题

**Reply by Orangenazuna** [reply]
嗯嗯，好的，我把配置文件发给你

**Comment by FidStyle** [comment]
![image](https://github.com/theme-shoka-x/hexo-theme-shokaX/assets/115857087/0092638b-efd4-440b-9ebd-4c809703a186)
_iconfont.styl是不用加这些前置代码的。不知道是不是这个问题
从i-steam开始即可
@Orangenazuna

**Reply by FidStyle** [reply]
稍等，应该不是这个问题（但是确实不需要加）

**Reply by FidStyle** [reply]
![image](https://github.com/theme-shoka-x/hexo-theme-shokaX/assets/115857087/620e1170-f464-4f6a-ab53-18da01671291)
这个地方写错了？你给我的压缩包是这样的
测试没问题。。

**Reply by FidStyle** [reply]
@Orangenazuna

**Comment by FidStyle** [comment, marked as answer]
文档中的`/source/css/_iconfont.styl`表意不明，已修改文档解决
